package_manifest.yaml

```diff
@@ -866,6 +866,11 @@ katello_packages:
     rubygem-pulp_file_client: {}
     rubygem-pulp_2to3_migration_client: {}
     rubygem-pulp_rpm_client: {}
+    rubygem-qpid_messaging:
```

Comment:
This was here before qpid_messaging was removed. Not sure if still relevant or correct